spec/features/vacation_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'RequestVacation', type: :feature do
+  let!(:authed_user) { create_logged_in_user }
+
+  it 'shows validation date for current location' do
+    visit new_vacation_path
+
+    within '#new_vacation' do
+      travel_to Time.zone.local(2023, 03, 31) do
+        Time.current.freeze
```

Comment:
I had to do it in order to trigger the :start_date error in a deterministic way. If I did it without stubbing time the test would fail on days close to a weekend.

Or do you mean line 11 (just found out it was unnecessary)?

Reply:
yes, line 11

Reply:
Fixed. Removed it.